app/models/student.rb

```diff
@@ -1,3 +1,11 @@
 class Student < ApplicationRecord
   belongs_to :instructor
+
+  def self.show_only_bool_true
+    where(returning_student: true)
+  end
+
+  def self.by_name
+    self.order(:name)
```

Comment:
I doubt it matters, but I remember Mike saying we don't actually have to specify 'self' in the method like the self.order.  I see you did leave that out in the method above.